libraries/lib-wave-track/WaveTrack.h

```diff
@@ -468,7 +484,13 @@ class WAVE_TRACK_API WaveTrack final
    /*!
     @pre `IsLeader()`
     */
-   void Split(double t0, double t1) /* not override */;
+   void Split(double t0, double t1);
+
+   /*!
+    @pre `IsLeader()`
+    */
+   std::pair<IntervalHolder, IntervalHolder> SplitAt(double t);
```

Comment:
I see the first returned `IntervalHolder` is never used. Also, I expect that the first `IntervalHolder`, if not null, points at existing clips? I find this would give even more strength to a simpler API that returns just one holder, to a _newly created object_, if any.

Reply:
No.  This API makes more sense to me, even if there is not yet any use for the first returned clip.  A split point defines two neighboring clips.